src/AngleSharp.Scripting.JavaScript/AngleSharp.Scripting.JavaScript.csproj

```diff
@@ -35,8 +35,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="jint, Version=2.10.4.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f, processorArchitecture=MSIL">
-      <HintPath>..\packages\jint.2.10.4\lib\portable40-net40+sl5+win8+wp8+wpa81\jint.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\..\..\..\src\packages\jint.2.10.4\lib\portable40-net40+sl5+win8+wp8+wpa81\jint.dll</HintPath>
```

Comment:
Sorry, this is because I pulled my fork as submodule somewhere and forgot to switch project.
Let's close this and I will follow-up with proper pull-requests when I get a chance, I have found more problems